app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -131,18 +131,28 @@ object GlobalAttributeTable {
     globalAttributes.list
   }
 
+  def toInt(s: Option[String]): Option[Int] = {
+    try {
+      Some(s.getOrElse("6").toInt)
```

Comment:
Instead of 6, could this actually be -1? Which is sort of a common number to use as a flag to mean something.